sw/device/tests/BUILD

```diff
@@ -2276,6 +2276,38 @@ opentitan_test(
     ],
 )
 
+opentitan_test(
+    name = "spi_device_smoketest",
+    srcs = ["spi_device_smoketest.c"],
+    cw310 = new_cw310_params(
+        otp = "//hw/ip/otp_ctrl/data/earlgrey_a0_skus/sival:otp_img_prod_manuf_personalized",
+        tags = ["hyper310"],
```

Comment:
Nit: the `fpga_cw310_test_rom` environment doesn't use hyper310, so this tag will be incorrect for that Bazel target

You can give different parameters for the two `exec_env`s by changing `None` to the name of the parameter (i.e. `cw310` here)

We might not need this `hyper310` tag at all though, since all sival tests will be hyper310 so we can filter by that